NuGet.Services.Staging.sln

```diff
@@ -30,6 +30,10 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Staging.Unit
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Logging", "src\NuGet.Services.Logging\NuGet.Services.Logging.xproj", "{3A70A370-5ABE-4A53-B5B9-297DB288964E}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Staging.EndToEndTests", "test\NuGet.Services.Staging.EndToEndTests\NuGet.Services.Staging.EndToEndTests.xproj", "{E6859EB8-303B-47C9-AF2B-E62398EDD37B}"

```

Comment:
Should we rename the test projects? E.g. `NuGet.Services.Staging.Tests.Common`, `NuGet.Services.Staging.Tests.EndToEnd`, ...